Websites/bugs.webkit.org/Bugzilla/Search.pm

```diff
@@ -2217,6 +2217,18 @@ sub SqlifyDate {
         my ($sec, $min, $hour, $mday, $month, $year, $wday)  = localtime($date);
         if ($sign && $sign eq '+') { $amount = -$amount; }
         $startof = 1 if $amount == 0;
+        elsif ($unit eq 'm') {
```

Comment:
I think that we usually add some marker around our changes, to make upgrades easier.

Reply:
Added